acts-as-taggable-on.gemspec

```diff
@@ -22,7 +21,7 @@ Gem::Specification.new do |gem|
     gem.post_install_message = File.read('UPGRADING.md')
   end
 
-  gem.add_runtime_dependency 'activerecord', '>= 6.0', '< 8'
+  gem.add_runtime_dependency 'activerecord', '>= 6.1', '< 7.2'
```

Comment:
hi there @seuros! Just out of curiosity, why did you end up changing the `< 8` to `< 7.2`?. 

Reading #1110, it seemed to me that the goal was to avoid needing a new release every time a new AR minor was released but I'm not sure if you see any other risks that could come from other "non major" releases.

Thanks!

Reply:
makes sense! thanks for the answer (and the release!)

Reply:
as predicted https://github.com/mbleigh/acts-as-taggable-on/issues/1130 Rails 7.2 released

Reply:
Let release a new version.